### PR TITLE
feat: Move "Contributing & Open Source" docs to top navbar

### PR DESCRIPTION
**Pull request by tomekpapiernik**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

## Further comments

Moved the docs that talk specifically about open source and contribution-related topics to top navbar. Adjusted sidebars to accommodate two separate sidebars. Added a top navbar button that allows to quickly return to the documentation.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -102,6 +102,18 @@ module.exports = {
...
         },
+        {
+          type: 'docSidebar',
+          position: 'left',
+          sidebarId: 'docs',
+          label: 'Documentation',
+        },
+        {
+          type: 'docSidebar',
+          position: 'right',
```
Comment (aeneasr) [inline docusaurus.config.js:113]: The right space is currently reserved for external links (github, slack, ...) while the left is for content :)

```suggestion
          position: 'left',
```

Comment (tomekpapiernik) [reply docusaurus.config.js]: Done!

**Review by vinckr — APPROVED** [review]
nice!

This declutters the docs while making the OSS stuff more prominent, really good solution 🙌 
I already have some ideas to improve the content there :-)

---

One small thing, 
do we need the "Home" route? 
I can't imagine a case where I want to get back to the "marketing" website. 
Also IMO the menu is missing a link to Ory Cloud Console. 
But we should address that in a different PR

**Comment by tomekpapiernik** [comment]
@vinckr That's a good point about the `Home` link. I don't really feel like we need it here. 
I'd be happy to delete it in this PR or in some future one. 

Also, adding the Ory Console link sounds like a good idea! This could help to promote Ory Cloud. 

@aeneasr are you ok with doing those 2 things in this PR? 
- Removing the `Home` link
- Adding `Ory Console` link to the `right` section

**Comment by aeneasr** [comment]
> Removing the Home link

How would one get to the home page? We should always have ways of navigating the page from anywhere to anywhere

**Comment by aeneasr** [comment]
Looks like there is no clear answer to this - products such as Google Cloud Docs always keep the "marketing" or product related info in the navigation: https://cloud.google.com/artifact-registry/docs

Other providers such as Stripe don't though: https://stripe.com/docs

Maybe we rename `Home` to `ory.sh` or `Website` and move it to the right to the other links next to slack, github, ...?

**Comment by tomekpapiernik** [comment]
Thanks for merging! We can come back to this in the future :)

**Pull request merged**